pvlib/iotools/pvgis.py

```diff
@@ -424,6 +448,12 @@ def get_pvgis_tmy(latitude, longitude, outputformat='json', usehorizon=True,
         base url of PVGIS API, append ``tmy`` to get TMY endpoint
     timeout : int, default 30
         time in seconds to wait for server response before timeout
+    utc_offset: int, default 0
+        Use to specify a timezone other than the default UTC zero. Will force
+        year to ``coerce_year`` if not zero.
+    coerce_year: int, default 1990
```

Comment:
```suggestion
    coerce_year: boolean
```

Reply:
doesn't match coerce year in other iotools which lets user force any year